### DISCUSSION TITLE
Missing spacing between multiple `CdsControlMessage` elements

### DISCUSSION THREAD

**Discussion by dorthrithil** [vmware/clarity#6451] [Q&A]
## Describe the bug

There is no spacing between the elements.

![Screenshot 2021-11-03 at 08 43 26](https://user-images.githubusercontent.com/3655915/140024223-c1129fc2-1838-4306-8797-e1d2cc115d1d.png)

## How to reproduce

https://stackblitz.com/edit/clarity-light-theme-v5-grtqze?file=src/app/app.component.html

## Expected behavior

Some padding between repeated `cds-control-message` elements. 

## Versions

**Clarity project:**

- [x] Clarity Core
- [X] Clarity Angular/UI

**Clarity version:**

- [X] v5.6.0

**Framework:**

- [X] Angular

**Framework version:**
_ie: Angular 12.0.2

**Comment by hippee-lee** [comment]
Applications should handle control message spacing with an appropriate spacing token on the elements that need it:

```html
        <cds-control-message cds-layout="m-t:sm"
          >Another message</cds-control-message
        >
```

I'm going to convert this to a Q&A discussion instead of closing the issue.

**Reply by dorthrithil** [reply]
The problem here is, that those would need to be applied conditionally if the messages are e.g. error messages which may not be visible all at once. The first message should not have spacing. The obvious solution would be to add a container with `cds-layout="vertical gap:sm"`, but a wrapper is not rendered by `cds-input` etc.

Adding code to control the spacing seems like a huge overhead in contrast to a simple wrapper with css.
